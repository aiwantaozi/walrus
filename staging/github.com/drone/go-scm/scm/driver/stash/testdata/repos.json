{
    "size": 25,
    "limit": 25,
    "isLastPage": false,
    "values": [
        {
            "slug": "my-repo",
            "id": 1,
            "name": "my-repo",
            "scmId": "git",
            "state": "AVAILABLE",
            "statusMessage": "Available",
            "forkable": true,
            "project": {
                "key": "PRJ",
                "id": 2,
                "name": "PRJ",
                "public": false,
                "type": "NORMAL",
                "links": {
                    "self": [
                        {
                            "href": "http://example.com:7990/projects/PRJ"
                        }
                    ]
                }
            },
            "public": false,
            "links": {
                "clone": [
                    {
                        "href": "ssh://git@example.com:7999/prj/my-repo.git",
                        "name": "ssh"
                    },
                    {
                        "href": "http://jcitizen@example.com:7990/scm/prj/my-repo.git",
                        "name": "http"
                    }
                ],
                "self": [
                    {
                        "href": "http://example.com:7990/projects/PRJ/repos/my-repo/browse"
                    }
                ]
            }
        },
        {
            "slug": "quux",
            "id": 2,
            "name": "quux",
            "scmId": "git",
            "state": "AVAILABLE",
            "statusMessage": "Available",
            "forkable": true,
            "project": {
                "key": "PRJ",
                "id": 2,
                "name": "different_name",
                "public": false,
                "type": "NORMAL",
                "links": {
                    "self": [
                        {
                            "href": "http://example.com:7990/projects/PRJ"
                        }
                    ]
                }
            },
            "public": false,
            "links": {
                "clone": [
                    {
                        "href": "ssh://git@example.com:7999/prj/quux.git",
                        "name": "ssh"
                    },
                    {
                        "href": "http://jcitizen@example.com:7990/scm/prj/quux.git",
                        "name": "http"
                    }
                ],
                "self": [
                    {
                        "href": "http://example.com:7990/projects/PRJ/repos/quux/browse"
                    }
                ]
            }
        }
    ],
    "start": 0
}