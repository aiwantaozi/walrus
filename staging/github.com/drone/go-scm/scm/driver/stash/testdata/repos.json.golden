[
    {
        "ID": "1",
        "Namespace": "PRJ",
        "Name": "my-repo",
        "Perm": null,
        "Branch": "master",
        "Private": true,
        "Clone": "http://example.com:7990/scm/prj/my-repo.git",
        "CloneSSH": "ssh://git@example.com:7999/prj/my-repo.git",
        "Link": "http://example.com:7990/projects/PRJ/repos/my-repo/browse",
        "Created": "0001-01-01T00:00:00Z",
        "Updated": "0001-01-01T00:00:00Z"
    },
   {
        "ID": "2",
        "Namespace": "PRJ",
        "Name": "quux",
        "Perm": null,
        "Branch": "master",
        "Private": true,
        "Clone": "http://example.com:7990/scm/prj/quux.git",
        "CloneSSH": "ssh://git@example.com:7999/prj/quux.git",
        "Link": "http://example.com:7990/projects/PRJ/repos/quux/browse",
        "Created": "0001-01-01T00:00:00Z",
        "Updated": "0001-01-01T00:00:00Z"
    }
]